ckanext/nextgeoss/public/nextgeoss.css

```diff
@@ -335,7 +335,7 @@ h4.title a {
   margin-bottom: 20px;
 }
 
-.filters-header .fa-filter {
+.filters-header .icon-filter {
```

Comment:
@kmbn this needs to be fa-filter instead of icon-filter. Because CKAN-2.7.2 uses Font Awesome 3, where icon is replaced with fa

Reply:
@JGulic You're right—I think there was a mixup with my CSS file. I didn't intend to revert back to `icon-` and I didn't intend to change the search layout either as you noted below.. I'll edit the CSS.